sdk/compute/Azure.ResourceManager.Compute/src/Customize/Models/UserInitiatedRedeploy.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.ResourceManager.Compute.Models
+{
+    public partial class UserInitiatedRedeploy
+    {
+        internal UserInitiatedRedeploy(bool? automaticallyApprove, string dummyProperty, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            AutomaticallyApprove = automaticallyApprove;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        private string DummyProperty { get; }
```

Comment:
let us add some comment here to describe why we have this otherwise we might forget what this is for.